samples/snippets/batch_process_documents_sample_v1beta3.py

```diff
@@ -38,6 +38,10 @@ def batch_process_documents(
     timeout: int = 300,
 ):
 
+    # You must set the api_endpoint if you use a location other than 'us', e.g.:
+    # client = documentai.DocumentProcessorServiceClient(
+    #     {"api_endpoint": "eu-documentai.googleapis.com"}
+    # )
     client = documentai.DocumentProcessorServiceClient()
```

Comment:
I generally dislike commented code because you can't test it or even guarantee it'll compile once uncommented. Would it be acceptable to do something like this instead? 
```suggestion
    # You must set the api_endpoint if you use a location other than 'us', e.g.:
    opts = {}
    if location == "eu":
       opts = {"api_endpoint": "eu-documentai.googleapis.com"}
    client = documentai.DocumentProcessorServiceClient(opts)
```

Reply:
Done